Utils/proxy.py

```diff
@@ -24,14 +23,21 @@
 import sys
 import os.path  # for checking if file is present or not
 import subprocess
+from os import getuid
+
+# run it as sudo
+if getuid() != 0:
+    print("error!")
+    sys.exit()
 
 apt_ = r'/etc/apt/apt.conf'
 apt_backup = r'./.backup_proxy/apt.txt'
 bash_ = r'/etc/bash.bashrc'
 bash_backup = r'./.backup_proxy/bash.txt'
 env_ = r'/etc/environment'
 env_backup = r'./.backup_proxy/env.txt'
-docker='/etc/systemd/system/docker.service.d/http-proxy.conf'
+docker_= r'/etc/systemd/system/docker.service.d/http-proxy.conf'
+docker_backup = r'./.backup_proxy/docker.txt'
```

Comment:
Generally this file not exists (because docker not installed).
Then no reason to do backup
If we want to remove proxy config this is enough to delete file

Reply:
I understand that it should be deleted with a file which does not originally exist.
It is the same for apt.conf.

If so,
2 :) Remove Proxy "
3 :) Restore Default
, It should be deleted normally.

It is not deleted in the current script.

So, I created an empty file first, so I backed it up.
As a result, I hoped that the empty file will be written back and restored when Restore.

Reply:
Also you need run demon reload for systemctl and restart docker daemon... Or just restart machine

Reply:
Thank you.
You are right.
I need to make corrections including other indications.